src/sentry/static/sentry/app/components/group/releaseStats.jsx

```diff
@@ -177,34 +219,33 @@ const GroupReleaseStats = createReactClass({
                 firstSeen={group.firstSeen}
                 lastSeen={group.lastSeen}
               />
-
               <h6>
                 <span>{t('First seen')}</span>
-                {environment.name && <small>({environment.name})</small>}
+                {environment ? <small>({environment.name})</small> : null}
               </h6>
 
               <SeenInfo
                 orgId={orgId}
                 projectId={projectId}
-                date={firstSeenEnv}
+                date={data.firstSeen}
                 dateGlobal={group.firstSeen}
                 hasRelease={hasRelease}
-                environment={environment.name}
+                environment={environment ? environment.name : null}
                 release={data.firstRelease ? data.firstRelease.release : null}
                 title={t('First seen')}
               />
 
               <h6>
                 <span>{t('Last seen')}</span>
-                {environment.name && <small>({environment.name})</small>}
+                {environment ? <small>({envName})</small> : null}
```

Comment:
For first seen it's `environment.name`, should they be the same?

Reply:
yes, thanks